File: FactuMex/Models/Valija.cs
using System;
using System.Collections.Generic;

namespace CallCenter.DBModel
{
    public partial class Valija
    {
        public long Idvalija { get; set; }
        public string Etiqueta { get; set; }
        public DateTime? Fecha { get; set; }
        public string Cb { get; set; }
        public string Cs { get; set; }
        public string Usuario { get; set; }
    }
}

File: FactuMex/ClientApp/src/app/valija/valija.ts
import { Factura } from "../factura/factura";

export class Notificacion {
    constructor(
        public Msj: string,
        public Src: string
    ) {
    }
}

export class Valija {
    constructor(
        public Idvalija: string,
        public Etiqueta: string,
        public Fecha: string,
        public Cb: string,
        public Cs: string,
        public Usuario: string
    ) {
    }
}
export class ValxFac {
    constructor(
        public val:Valija,
        public fac:Factura,
        public actn:string
    ) {
    }
}